### PR TITLE
Update Akka and sbtNativePackager

### PR DESCRIPTION
**Pull request by mkurz**
https://akka.io/blog/news/2019/02/13/akka-2.5.21-released

https://github.com/sbt/sbt-native-packager/blob/master/CHANGELOG.md
https://github.com/sbt/sbt-native-packager/compare/v1.3.14...v1.3.18

### CODE REVIEW

**Comment by marcospereira** [comment]
@mkurz there are now some conflicts here since #9023 was merged first.

**Comment by dwijnand** [comment]
Probably best wait for https://github.com/playframework/playframework/pull/9026 to land before putting in the effort.

**Comment by mkurz** [comment]
This should turn green now.
I also included a bug fix release of hibernate-validator:
http://in.relation.to/2019/02/18/hibernate-validator-6015-final-out/

File: project/plugins.sbt [hunk 1]
```diff
@@ -5,7 +5,7 @@ enablePlugins(BuildInfoPlugin)
 val Versions = new {
   // when updating sbtNativePackager version, be sure to also update the documentation links in
```
Comment (PromanSEW) [inline project/plugins.sbt]: You did not update version according this comment

Comment (mkurz) [reply project/plugins.sbt]: Problem is the docs are not published for 1.3.18 (yet):
https://sbt-native-packager.readthedocs.io/en/v1.3.18/
Same for v1.3.17 and v1.3.16...
image

Comment (ignasi35) [reply project/plugins.sbt]: related https://github.com/sbt/sbt-native-packager/issues/1087

Comment (ignasi35) [reply project/plugins.sbt]: raised https://github.com/sbt/sbt-native-packager/issues/1203

Comment (mkurz) [reply project/plugins.sbt]: So should we wait for the docs to be published before merging this pull request here?

Comment (dwijnand) [reply project/plugins.sbt]: No, I think we can track that issue Ignasi opened, and land this for now.

Comment (mkurz) [reply project/plugins.sbt]: :+1: If you could backport as well that would be nice :cake:

**Comment by dwijnand** [comment]
2.7.x: 067a95071b04900580adb917ed8cdced44131a2c

File: project/plugins.sbt (hunk 1 above)
Comment (mkurz) [reply project/plugins.sbt]: @ignasi35 #9053

**Pull request merged**